packages/core/src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -151,8 +142,8 @@ describe('UserCredentialsUtils', function () {
                 `creds.secretKey: "${profile.aws_access_key_id}" !== "${creds.secretKey}"`
             )
 
-            await assertDoesNotThrow(async () => await fs.checkPerms(credentialsFilename, 'r--'))
-            await assertDoesNotThrow(async () => await fs.checkPerms(credentialsFilename, '-w-'))
+            await assert.doesNotReject(async () => await fs.checkPerms(credentialsFilename, 'r--'))
```

Comment:
Are these changes somehow related to this PR? Otherwise the change looks good to me

Reply:
no just a fixup for a recent commit